pomm-project/pomm-bundle/2.0/manifest.json

```diff
@@ -0,0 +1,12 @@
+{
+    "alias": "pomm",
```

Comment:
I've added this feature on my todo-list, I thought I implemented that already but that's not the case apparently (keep in mind that everything is in alpha mode for now). I won't be able to use symfony/config as the server is not written in PHP :)